src/protocol/logpoint.ts

```diff
@@ -123,7 +124,13 @@ client.Analysis.addAnalysisPointsListener(({ analysisId, points }) => {
       LogpointHandlers.onPointLoading!(logGroupId, point, time, location);
     });
   }
-});
+};
+
+export function setupLogpoints() {
+  client.Analysis.addAnalysisResultListener(({ analysisId, results }) => onAnalysisResult);
```

Comment:
shouldn't this be?

```js
client.Analysis.addAnalysisResultListener(({ analysisId, results }) => onAnalysisResult(analysisId, results))
```

Reply:
you're absolutely right, i'll start leaving late changes for the morning so i don't miss things like this.